geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseQueryPlan.scala

```diff
@@ -54,13 +55,17 @@ case class EmptyPlan(filter: HBaseFilterStrategyType) extends HBaseQueryPlan {
   override def scan(ds: HBaseDataStore): CloseableIterator[SimpleFeature] = CloseableIterator.empty
 }
 
-case class ScanPlan(filter: HBaseFilterStrategyType,
+case class ScanPlan(sft: SimpleFeatureType,
+                    filter: HBaseFilterStrategyType,
                     table: TableName,
                     ranges: Seq[Scan],
                     clientSideFilter: Option[Filter],
                     entriesToFeatures: Iterator[Result] => Iterator[SimpleFeature]) extends HBaseQueryPlan {
   override def scan(ds: HBaseDataStore): CloseableIterator[SimpleFeature] = {
-    val results = new BatchScan(ds.connection, table, ranges, ds.config.queryThreads, 100000)
+    val remoteFilters = filter.filter.map { filter =>
+      new JSimpleFeatureFilter(sft, filter)
+    }.toSeq
+    val results = new BatchScan(ds.connection, table, ranges, ds.config.queryThreads, 100000, remoteFilters)
```

Comment:
These are changes that are pending in a different PR. I think @jnh5y mentioned to keep these changes separate and only push JSimpleFilter.